composer.json

```diff
@@ -1,14 +1,77 @@
 {
     "name": "ec-europa/subsite",
-    "description": "Subsite template to initiate a new subsite project that uses toolkit.",
-    "homepage": "https://github.com/ec-europa/toolkit#nexteuropa-toolkit",
-    "license": "EUPL-1.1",
+    "description": "Project template for NextEuropa Subsites on ssk 4.x",
```

Comment:
Improve description, suggestion:
`Drupal 8 template for websites hosted in DIGIT`